orangecontrib/spectroscopy/tests/test_readers.py

```diff
@@ -442,25 +442,43 @@ class TestNea(unittest.TestCase):
 
     def test_open_v1(self):
         data = Orange.data.Table("spectra20_small.nea")
-        self.assertEqual(len(data), 12)
-        self.assertEqual("channel", data.domain.metas[2].name)
-        np.testing.assert_almost_equal(getx(data), [0.000295, 0.00092])
-        self.assertEqual("O0A", data.metas[0][2])
-        np.testing.assert_almost_equal(data.X[0, 0], 10.2608052)  # O0A
-        self.assertEqual("O0P", data.metas[6][2])
-        np.testing.assert_almost_equal(data.X[6, 0], 0)  # O0P
+        self.assertEqual(len(data), 20)
+        self.assertEqual("run", data.domain.metas[2].name)
+        self.assertEqual("channel", data.domain.metas[3].name)
```

Comment:
Also, I printed out the table, and "Row" and "Column" are read as (-0.5, -0.5), even though there is only (0, 0) in the file. Why did this get changed?

The fixed version of this tests should also test if "Row" and "Column" were read correctly.